rpm/SPECS/iotaul.spec

```
@@ -170,6 +170,17 @@ fi
 %{_install_dir}
 
 %changelog
+* Mon Feb 3 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.20.0
+- Add: allow handle binary messages (#530)
+- Fix: remove request obsolete library, using iotagent-node-lib.request instead (iotagent-node-lib#858)
+- FIX: default mqtt keepalive value by conf (must be 60 instead of 0) (#527)
+- FIX: provide device type to findConfiguration to achieve a better group match in getEffectiveApiKey (iota-node-lib#1155)
+- FIX: update polling when device is updated by adding endpoint (needs iota-node-lib >= 2.19) (twin for iota-json#602)
+- FIX: Remove preprocess stripping of explicitAttrs (iotagent-node-lib#1151)
+- FIX: Add graceful shutdown listening to SIGINT (#524)
+- Upgrade logops dep from 2.1.0 to 2.1.2 due to colors dependency corruption
+- Upgrade iotagent-node-lib dependency from 2.18.0 to 2.19.0
```

Fixing date and a little bit of cleanup:

```suggestion
* Mon Feb 7 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.20.0
- Add: allow to handle binary messages (#530)
- Fix: default mqtt keepalive value by conf (must be 60 instead of 0) (#527)
- Fix: provide device type to findConfiguration to achieve a better group match in getEffectiveApiKey (iota-node-lib#1155)
- Fix: update polling when device is updated by adding endpoint (needs iota-node-lib >= 2.19) (twin for iota-json#602)
- Fix: remove preprocess stripping of explicitAttrs (iotagent-node-lib#1151)
- Fix: add graceful shutdown listening to SIGINT (#524)
- Fix: remove request obsolete library, using iotagent-node-lib.request instead (iotagent-node-lib#858)
- Upgrade logops dep from 2.1.0 to 2.1.2 due to colors dependency corruption
- Upgrade iotagent-node-lib dependency from 2.18.0 to 2.19.0
```